blueprint/user-libraries/library-entries.apib

```diff
@@ -2,3 +2,35 @@
 
 + Parameters
     + id (number, `1`) - ID of the library entry resource
+
++ Attributes (libraryEntriesAttributes)
+
+### GET List [GET /anime-characters]
+
++ Response 200 (application/vnd.api+json)
+    + Attributes
+        + data (array, required)
+            + (libraryEntries)
+        + meta (object, required)
+            + count: 0 (number, required) 
+            <!-- Because of some errors with NSFW entries-->
```

Comment:
❌ MSON doesn't have comments of any kind - this will display `<!-- ... -->` literally in the response attributes